pvlib/iotools/crn.py

```diff
@@ -103,19 +110,13 @@ def read_crn(filename):
     dtindex = pd.to_datetime(dts['UTC_DATE'] + dts['UTC_TIME'].str.zfill(4),
                              format='%Y%m%d%H%M', utc=True)
     data = data.set_index(dtindex)
-    try:
-        # to_datetime(utc=True) does not work in older versions of pandas
-        data = data.tz_localize('UTC')
-    except TypeError:
-        pass
```

Comment:
Too bad I neglected to indicate the versions that needed this. I *think* the tests are comprehensive enough that we can safely remove this if they still pass.